src/webgl/p5.Camera.js

```diff
@@ -1720,23 +1720,19 @@ p5.Camera.prototype._orbit = function(dTheta, dPhi, dRadius) {
   let camTheta = Math.atan2(diffX, diffZ); // equatorial angle
   let camPhi = Math.acos(Math.max(-1, Math.min(1, diffY / camRadius))); // polar angle
 
-  // add change
-  camTheta += dTheta;
-  camPhi += dPhi;
-  camRadius += dRadius;
+  // add change according to the direction of this.upY
+  let directionflag = this.upY > 0 ? 1 : -1;
+  camTheta += directionflag*dTheta;
+  camPhi += directionflag*dPhi;
+  if (camPhi <= 0 || camPhi>=Math.PI) {
```

Comment:
Super minor, but on this line and the ones above, can we add spaces around the operators so they match the style of the rest of the file?